packages/flutter_test/lib/src/test_compat.dart

```diff
@@ -21,16 +21,16 @@ import 'package:test_api/src/backend/state.dart'; // ignore: implementation_impo
 // ignore: deprecated_member_use
 import 'package:test_api/test_api.dart';
 
-Declarer _localDeclarer;
+Declarer? _localDeclarer;
 Declarer get _declarer {
-  final Declarer declarer = Zone.current[#test.declarer] as Declarer;
+  final Declarer? declarer = Zone.current[#test.declarer] as Declarer?;
   if (declarer != null) {
     return declarer;
   }
   // If no declarer is defined, this test is being run via `flutter run -t test_file.dart`.
   if (_localDeclarer == null) {
     _localDeclarer = Declarer();
-    Future<void>(() {
+    Future<dynamic>(() {
```

Comment:
Why is this no longer `Future<void>`? It doesn't appear to return anything.

Reply:
Changed this back to void. I originally made this dynamic because the analyzer was complaining. Turns out, that was a bug in the analyzer: https://github.com/dart-lang/linter/issues/2185